package.json

```diff
@@ -41,22 +41,22 @@
     "json-loader": "^0.5.7",
     "lodash": "^4.17.15",
     "mini-css-extract-plugin": "^2.2.0",
-    "modulejs": "2.2.0",
-    "moment": "2.27.0",
+    "modulejs": "2.9.0",
```

Comment:
be careful with that, it's an old lib with no good reason to use in the world of ES modules